src/includes/class-alma-wc-payment-gateway.php

```diff
@@ -557,7 +557,10 @@ private function get_plan_upon_trigger_display_text( $plan_index ) {
 		if ( 1 === $plan_index ) {
 			return Alma_WC_Payment_Upon_Trigger::get_display_text();
 		}
-		return sprintf( '%s %s', $plan_index - 1, _n( 'month', 'months', $plan_index - 1, 'alma-woocommerce-gateway' ) );
+		// translators: 'In' refers to a date or time, like in 'In two weeks' or 'In three hours'.
+		$in = __( 'In', 'alma-woocommerce-gateway' );
```

Comment:
I am not fan of "words only" translations and prefer that translations refer to entire phrases like this diff purposed bellow.

KISS 😉